__tests__/istanbul-linker-util.ts

```diff
@@ -1,18 +1,24 @@
-const { parse } = require('acorn');
-const { getFileCoverageDataByName } = require('istanbul-spy');
+import { parse } from 'acorn';
+import { getFileCoverageDataByName } from 'istanbul-spy';
 
 // Istanbul.js, why did you make me do this, I trusted you...
 // This would have probably been a whole lot easier with regex, maybe, but probably not as clear.
 // Also, this is PURELY for testing
 
 // hah! damn...
 
+type TSettings = {
+  onIstanbulCoverageVariable: (name: string) => any
+};
+
 class Recurse {
-  constructor(settings) {
+  settings: TSettings;
+
+  constructor(settings: TSettings) {
     this.settings = settings;
   }
 
-  into(ast) {
+  into(ast: Node | any) {
```

Comment:
I this `ast` should be optional here with type `Node`.

Reply:
There are no properties like 'type' existing on Node. In general istanbul is not even a typed library so this file should not be converted to TypeScript at all but if it is then `any` is what we have to use.